stock_picking_progress/hooks.py

```diff
@@ -32,25 +31,29 @@ def setup_move_line_progress(cr):
         )
     ]
     openupgrade.add_fields(env, field_spec)
-    logger.info("filling up %s on %s", column, table)
+    logger.info(f"filling up {column} on {table}")
     cr.execute("""UPDATE stock_move_line SET progress =0 WHERE state = 'cancel'""")
     cr.execute(
         """
-        UPDATE stock_move_line SET progress =(qty_done / reserved_qty) * 100
-        WHERE state not in ('done', 'cancel') AND  reserved_qty > 0.0
+        UPDATE stock_move_line sml
+        SET progress =(sml.quantity / sm.product_uom_qty) * 100
```

Comment:
This is not quite good.

As several move lines can be concerned by one stock move. So, this is the total amount of quantities of stock move lines / sm.product_qty